src/vic3_world/provinces/vic3_province_definitions_loader.cpp

```diff
@@ -0,0 +1,31 @@
+#include "src/vic3_world/provinces/vic3_province_definitions_loader.h"
+
+#include "external/commonItems/CommonRegexes.h"
+#include "external/commonItems/ParserHelpers.h"
+
+
+
+vic3::ProvinceDefinitionsLoader::ProvinceDefinitionsLoader()
+{
+   parser_.registerRegex("x[0-9A-F]{6}", [this](const std::string& province_color, std::istream& input_stream) {
```

Comment:
Lowercase hex letters are legit and used elsewhere. I know they autogenerate province terrain file but mods won't.